knimin/lib/data_access.py

```diff
@@ -489,7 +490,7 @@ def _months_between_dates(self, d1, d2):
         # Calculate the number of 12-month periods between the years
         return (d2.year - d1.year) * 12 + (d2.month - d1.month)
 
-    def format_survey_data(self, md, external_surveys=None):  # noqa
+    def format_survey_data(self, md, external_surveys=None, full=False):  # noqa
```

Comment:
Again, this is code that will be refactored as soon as it works and we can add tests, so documentation at this point is moot.